scripts/makeLegacyPlots_postFit.py

```diff
@@ -0,0 +1,935 @@
+            chBox.SetNDC()
+            chBox.SetTextSize(cmsTextSize+20)
+            chBox.SetTextFont(43)
+            chBox.SetTextColor(kBlack)
+            chBox.SetTextAlign(13)
+            
+    CMSbox.Draw()
+    extraTextBox.Draw()
+    lumibox.Draw()
+    
+    if args.legend: leg.Draw()
+    if chBox: chBox.Draw()
+
+    if not args.name:
+            if "1b1j" in args.sel:      selName = "res1b"
+            if "2b0j" in args.sel:      selName = "res2b"
+            if "boosted" in args.sel:   selName = "boosted"
+            if "GGFclass" in args.sel:  selName = "classGGF"
+            if "VBFclass" in args.sel:  selName = "classVBF"
+            if "DYclass" in args.sel:   selName = "classDY"
+            if "ttHclass" in args.sel:  selName = "classttH"
+            if "TTclass" in args.sel:   selName = "classTT"
+    else: selName = args.name
+
+    selBox = TLatex  (l + 0.04 , 1 - t - 0.02 - 0.06, selName)
+    selBox.SetNDC()
+    selBox.SetTextSize(cmsTextSize+20)
+    selBox.SetTextFont(43)
+    selBox.SetTextColor(kBlack)
+    selBox.SetTextAlign(13)
+    selBox.Draw()
+
+    ###################### RATIO PLOT #################################
+    if args.ratio:
+        bkgStack.GetXaxis().SetTitleSize(0.00)
+        bkgStack.GetXaxis().SetLabelSize(0.00)
+        bkgSumNS.GetXaxis().SetLimits(0,binNumbs)
+
+        c1.cd()
+        #pad2 = TPad ("pad2", "pad2", 0, 0.0, 1, 0.2496)
+        pad2 = TPad ("pad2", "pad2", 0, 0.0, 1, 0.3096)
+        pad2.SetLeftMargin(0.12)
+        pad2.SetTopMargin(0.045)
+        pad2.SetBottomMargin(0.4)
+        pad2.SetGridy(True)
+        pad2.SetFrameLineWidth(3)
+        pad2.Draw()
+        pad2.cd()
+
+        grRatio = makeDataOverMCRatioPlot(gDataNonScaled, hBkgEnvelopeNS, "grRatio")
+
+        bkgSumNS.GetXaxis().SetTitleFont(43) # so that size is in pixels
+        bkgSumNS.GetYaxis().SetTitleFont(43) # so that size is in pixels
+        bkgSumNS.GetXaxis().SetLabelFont(43) # so that size is in pixels
+        bkgSumNS.GetYaxis().SetLabelFont(43) # so that size is in pixels
+        bkgSumNS.GetYaxis().SetNdivisions(505)
+
+        #bkgSumNS.GetXaxis().SetTitle(bkgStack.GetXaxis().GetName())
+        bkgSumNS.SetTitle(plotTitle)
+        bkgSumNS.GetYaxis().SetTitle ("Data/Bkg.") #("Data/MC")
+        if args.label: bkgSumNS.GetXaxis().SetTitle (args.label)
+        else: bkgSumNS.GetXaxis().SetTitle (args.var)
+        bkgSumNS.GetXaxis().SetTitleOffset(3.9)
+        bkgSumNS.GetYaxis().SetTitleOffset(1.2)
+
+        bkgSumNS.GetXaxis().SetTitleSize(titleSize)
+        bkgSumNS.GetXaxis().SetLabelSize(labelSize)
+        bkgSumNS.GetYaxis().SetTitleSize(titleSize)
+        bkgSumNS.GetYaxis().SetLabelSize(labelSize)
+
+        bkgSumNS.GetXaxis().SetTickSize(0.10)
+        bkgSumNS.GetYaxis().SetTickSize(0.05)
+
+        bkgSumNS.SetStats(0)
+
+        removeEmptyPoints(grRatio) # commented since the blinding has been moved to makeDataOverMCRatioPlot
```

Comment:
In the original plotter this is commented out because, as the comment says the blinding is done in makeDataOverMCRatioPlot. 
In this script I don't see the blinding anywhere...

Reply:
these are postfit plots so no blinding will ever be needed, so I just took all the blinding parts away

Reply:
ok makes sense! So let's remove the comment!

Reply:
plus that is actually to be commented out cause in the meeting discussion some time ago we said that we want to keep also the points for data=0.0

Reply:
ok then let's comment it out!